Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -26,10 +27,18 @@ public class Platform : BindableObject, IPlatform, INavigation, IDisposable
 		bool _appeared;
 
 		bool _disposed;
+		UIView _pageOverlayView;
 
 		internal Platform()
 		{
-			_renderer = new PlatformRenderer(this);
+			_renderer = new PlatformRenderer(this)
+			{
+				View =
+				{
+					BackgroundColor = UIColor.White,
+					ContentMode = UIViewContentMode.Redraw
+				}
+			};
```

Comment:
Left from changes to `ViewWillAppear`. These can be set once during initialization.